tests/framework/extensions/provisioning/verify.go

```diff
@@ -93,7 +93,7 @@ func VerifyRKE1Cluster(t *testing.T, client *rancher.Client, clustersConfig *clu
 }
 
 // VerifyCluster validates that a non-rke1 cluster and its resources are in a good state, matching a given config.
-func VerifyCluster(t *testing.T, client *rancher.Client, cluster *steveV1.SteveAPIObject) {
+func VerifyCluster(t *testing.T, client *rancher.Client, cluster *steveV1.SteveAPIObject, clustersConfig *provisioninginput.Config) {
```

Comment:
What's the need to add `clustersConfig` to this function? Doesn't seem like anything is referencing it, so can probably be removed.

Reply:
agreed

Reply:
Isn't this where the sshTests value is stored?

Reply:
i think you are passing in clusterConfig as a param, which points to provisioninginput, but in the `VerifyCluster` function, you're referencing provisioninginput directly (via the import), and never actually reference the clusterConfig parameter, so i agree w/ Markus + Caleb, that this param doesn't seem neccessary :)